Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/EffectData.xml

```diff
@@ -7772,6 +7772,57 @@
         <Player Value="Caster"/>
         <Target Effect="AP_ReleaseInterceptorsSet" Value="TargetPoint"/>
     </CEffectIssueOrder>
+    <CEffectDamage id="AP_CarrierTaldarimLaserDamage" parent="DU_WEAP">
+        <ValidatorArray value="noMarkers"/>
+        <EditorCategories value="Race:Protoss"/>
+        <Kind value="Ranged"/>
+        <KindSplash value="Splash"/>
+        <Amount value="55"/>
+        <Death value="Fire"/>
+    </CEffectDamage>
+    <CEffectEnumArea id="AP_CarrierTaldarimLaserDamageArea">
+        <EditorCategories value="Race:Protoss"/>
+        <ExcludeArray Value="Target"/>
+        <IncludeArray Effect="AP_CarrierTaldarimLaserCP" Value="Target"/>

```

Comment:
Not certain about this; should this be AP_CarrierTaldarimLaserEffect? (or maybe there should be a second entry?) It looks to me like it's including the target of the regular version, but not the Squadsight version